src/components/Settings/AccountTab.react.js

```diff
@@ -372,7 +372,7 @@ class AccountTab extends React.Component {
             <div>
               <Select
                 onChange={this.handleAvatarTypeChange}
-                value={avatarType}
+                value={avatarType ? avatarType : 'default'}
```

Comment:
Try to change the default avatarType value in the reducer level, so that it can reflect here.